rakelib/form526.rake

```diff
@@ -326,11 +331,9 @@ namespace :form526 do
         dis_translation_hash = get_dis_translation_hash(disability_array)
 
         treatment_facilities_transformed = fix_treatment_facilities_disability_name(form_data_hash,
-                                                                                    dis_translation_hash,
-                                                                                    disability_array)
+                                                                                    dis_translation_hash)
```

Comment:
This is why I advocate for adding a new line after and before the open/closing parens when going multi line. Only two spaces if indentation needed instead of 80.

(No change required, just a nit if you want to pursue.)